sunpy/data/data_manager/tests/conftest.py

```diff
@@ -0,0 +1,66 @@
+import shutil
+import tempfile
+import csv
+from unittest import mock
+
+import pytest
+from sunpy.data.data_manager.tests import mocks
+
+from sunpy.data.data_manager.cache import Cache
+from sunpy.data.data_manager.manager import DataManager
+from sunpy.data.data_manager.storage import InMemStorage, SqliteStorage
+
+DB_TESTDATA_FILE = 'sunpy/data/data_manager/tests/db_testdata.csv'
```

Comment:
We need to make sure this is going to be included in the sdist, you will need to add it to the `setup.cfg` file.